### PR TITLE
feat(api-client,core): Set availability status (WEBFOUND-50)

### PR DESCRIPTION
**Pull request by ffflorian**
## Pull Request Checklist

- [x] My code is covered by tests
- [x] I will [merge the PR as breaking change](https://github.com/wireapp/wire-web-packages/wiki/Releases#create-major-release), if the API contract changes

### CODE REVIEW

File: packages/core/src/main/broadcast/BroadcastService.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    const recipients = await this.cryptographyService.encrypt(plainTextArray, preKeyBundle);
+
+    return this.sendOTRBroadcastMessage(clientId, recipients, plainTextArray);
+  }
+
+  private async sendOTRBroadcastMessage(
+    sendingClientId: string,
+    recipients: OTRRecipients,
+    plainTextArray: Uint8Array,
+    data?: any
```
Comment (bennyn) [inline packages/core/src/main/broadcast/BroadcastService.ts]: Can this be specified?

Comment (ffflorian) [reply packages/core/src/main/broadcast/BroadcastService.ts]: Unfortunately not, we also have this in [ConversationService.ts#L479](https://github.com/wireapp/wire-web-packages/blob/master/packages/core/src/main/conversation/ConversationService.ts#L479).

File: packages/core/src/main/user/UserService.test.node.js [hunk 2]
```diff
@@ -1,29 +1,120 @@
...
       })
       .persist();
+
+    nock(Backend.PRODUCTION.rest)
+      .post(BroadcastAPI.URL.BROADCAST, body => {
+        hasState[body.sender] = true;
+        return true;
+      })
+      .query(() => true)
+      .reply(StatusCode.OK, '{}')
```
Comment (bennyn) [inline packages/core/src/main/user/UserService.test.node.js]: Let's return proper data here.

File: packages/core/src/main/user/UserService.test.node.js [hunk 3]
```diff
@@ -24,6 +51,71 @@ describe('UserService', () => {
...
+          ],
+          user: requestedUserId,
+        };
+
+        return [StatusCode.OK, JSON.stringify(data)];
+      })
+      .persist();
+  });
+
+  beforeEach(() => {
```
Comment (bennyn) [inline packages/core/src/main/user/UserService.test.node.js]: one-liner

File: packages/core/src/main/user/UserService.test.node.js [hunk 4]
```diff
@@ -1,18 +1,45 @@
...
 
 const PayloadHelper = require('../test/PayloadHelper');
 
-const HTTPS_URL = Backend.PRODUCTION.rest;
-
 describe('UserService', () => {
+  let hasState;
+
+  afterAll(() => {
+    nock.cleanAll();
```
Comment (bennyn) [inline packages/core/src/main/user/UserService.test.node.js]: one-liner

**Pull request merged**